missilewars-plugin/src/main/java/de/butzlabben/missilewars/MissileWars.java

```diff
@@ -120,6 +121,11 @@ public void onEnable() {
             PreFetcher.preFetchPlayers(new StatsFetcher(new Date(0L), ""));
         }
 
+        // Small check to make sure that PlaceholderAPI is installed
+        if(Bukkit.getPluginManager().getPlugin("PlaceholderAPI") != null) {
```

Comment:
Formatting (space between if and brackets). Please always format files before commiting (Strg + Alt + L on IntelliJ)

```suggestion
        if (Bukkit.getPluginManager().getPlugin("PlaceholderAPI") != null) {
```